cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -153,7 +153,7 @@ public static void writeStartId(String className, String id, boolean writeMaster
     public static void writeStartIdName(String className, String id, String name, String cimNamespace, XMLStreamWriter writer, CgmesExportContext context) throws XMLStreamException {
         writeStartId(className, id, true, cimNamespace, writer, context);
         writer.writeStartElement(cimNamespace, CgmesNames.NAME);
-        writer.writeCharacters(name);
+        writer.writeCharacters(name.length() > 32 ? name.substring(0, 32) : name); // name should not be longer than 32 characters
```

Comment:
I found empty names importing a real-world test case that would result in NPE, we should protect the check for length. Anyway, I have to check the import for that case

Reply:
I don't agree, name is here always from `nameOrId` in the code, it should never be empty (when name is empty, id is returned)

Reply:
The NPE is raised when writing geographical region names. I will debug it.

Reply:
cf e42901bf348ae8d35f5b606abc1a3308f6b5ff1c

Reply:
Your fix in commit e42901b works ok. Sub-regions with duplicated names are preserved, although exported without name.

Maybe we could solve the handling of regions and sub-regions with duplicated names in a separate PR.